app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterMovementReferenceNumberController.scala

```diff
@@ -60,15 +60,20 @@ class EnterMovementReferenceNumberController @Inject() (
   import cats.data.EitherT._
   implicit val dataExtractor: DraftC285Claim => Option[MovementReferenceNumber] = _.movementReferenceNumber
 
+  def enterBulkMrn(): Action[AnyContent]  = changeOrEnterMrn(false)
+  def changeBulkMrn(): Action[AnyContent] = changeOrEnterMrn(true)
+
   def enterMrn(): Action[AnyContent]  = changeOrEnterMrn(false)
   def changeMrn(): Action[AnyContent] = changeOrEnterMrn(true)
 
   protected def changeOrEnterMrn(isAmend: Boolean): Action[AnyContent] =
     authenticatedActionWithSessionData.async { implicit request =>
-      withAnswers[MovementReferenceNumber] { (_, previousAnswer) =>
+      withAnswers[MovementReferenceNumber] { (fillingOutClaim, previousAnswer) =>
+        val router    =
+          getRoutes(getNumberOfClaims(fillingOutClaim.draftClaim), Some(MovementReferenceNumber(Right(MRN("")))))
```

Comment:
Why can't you just pass None here for the entry number?

Reply:
Because then I get the wrong Router object back:
```case object DefaultRoutes extends MRNRoutes with SingleRoutes```